tests/common.py

```diff
@@ -19,13 +19,26 @@
 CONAN_GIF_VIDEO_URL = "https://samples.clarifai.com/3o6gb3kkXfLvdKEZs4.gif"
 TOY_VIDEO_FILE_PATH = os.path.dirname(__file__) + "/assets/toy.mp4"
 
-ENGLISH_TEXT = (
-    "My spanish test is tomorrow morning. I don't feel like studying tonight, but I must study."
-)
-SPANISH_TEXT = "No me apetece nada estudiar esta noche."
-
 ENGLISH_AUDIO_URL = "https://samples.clarifai.com/english_audio_sample.mp3"
 
+TRANSLATION_TEST_DATA = {
```

Comment:
Given these are specific to the model predict test, maybe it shouldn't be in tests/common ?

I see there's other things in this file that aren't common either, but just staying within scope

Reply:
yeah I see.
Do you suggest that I move them to the test_public_models_script or perhaps a separate file?

Reply:
I think either moving them into test_public_models_script would be good, because it has to import all of them anyways

Reply:
sure let me do that

Reply:
I confirm all tests pass properly. But we will add them slowly to make sure all is scalable. @HarmitMinhas96  see [here](https://github.com/Clarifai/clarifai-python-grpc/blob/eagle2050-rebased/tests/public_models/test_public_models_predicts.py#L394) and [here](https://github.com/Clarifai/clarifai-python-grpc/blob/eagle2050-rebased/tests/public_models/test_public_models_predicts.py#L419) 

Are you cool with that @HarmitMinhas96  ? I will create tickets for those two tests so we add one each week.

I will also let Marco know what to synchronize across dev/staging/prod